h2o-extensions/target-encoder/src/main/java/hex/schemas/TargetEncoderTransformParametersV3.java

```diff
@@ -13,7 +13,7 @@
   public KeyV3.ModelKeyV3<TargetEncoderModel> model;
   @API(help = "Seed value")
   public long seed;
-  @API(help = "Data leakage handling strategy. Default to None.", values = {"None", "KFold", "LeaveOneOut"})
+  @API(help = "Data leakage handling strategy.", values = {"None", "KFold", "LeaveOneOut"})
```

Comment:
@deil87 FYI: there's possibility to use `valuesProvider=LeakageStrategyHandler.class` if you want this to be updated automatically when adding a new strategy:
you just need to have `LeakageStrategyHandler` to extend `EnumValuesProvider` (cf. other examples).

Reply:
@seb-h2o thanks! will definitely try this approach